src/state/predictions/index.ts

```diff
@@ -154,14 +149,37 @@ export const fetchRounds = createAsyncThunk<{ [key: string]: ReduxNodeRound }, n
         [reduxNodeRound.epoch.toString()]: reduxNodeRound,
       }
     }, {})
+
+    const publicData = {
+      status,
+      currentEpoch,
+      intervalSeconds,
+      minBetAmount,
+      rounds: roundData,
+      ledgers: {},
+      claimableStatuses: {},
+    }
+
+    if (!account) {
+      return publicData
+    }
+
+    const epochs =
+      currentEpoch > PAST_ROUND_COUNT ? range(currentEpoch, currentEpoch - PAST_ROUND_COUNT) : [currentEpoch]
```

Comment:
<img width="543" alt="Screen Shot 2022-04-06 at 22 07 10" src="https://user-images.githubusercontent.com/97418926/162099890-3b47e0e2-96a5-41cd-b051-d249d8122def.png">
The old logic for range is `earliestEpoch -> currentEpoch + 1`. I'm wondering why they need to fetch future epoch here

Reply:
Because we had an increasing range that was excluding the latest element that is why it was used like that. Now we have a decreasing range so we don't need it anymore.

Reply:
Got it. Thanks